cmd/srcd-server/engine/sql.go

```diff
@@ -101,20 +101,28 @@ func createGitbase(opts ...docker.ConfigOption) docker.StartFunc {
 		}
 
 		// gitbase can be CPU intensive, set a limit of 75% of the host CPU.
-		// Docker option --cpus is not available in the api, but it is a shorthand.
-		// --cpus="1.5" is equivalent to --cpu-period="100000" --cpu-quota="150000"
 
-		ncpu, err := docker.NCPU(ctx)
+		host := &container.HostConfig{}
+		desktop, err := docker.IsDockerDesktop(ctx)
```

Comment:
`Server` already has `hostOS` options.

Reply:
oh, I missed that config. Done in b5159dc, will rebase before merging.